### PR TITLE
Events list widget view more link

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [TEC-3600]

This PR adds templates and styles for the view more link in the events list widget.

**Artifact:**

image

[TEC-3600]: https://moderntribe.atlassian.net/browse/TEC-3600

### CODE REVIEW

File: src/views/v2/widgets/events-list.php [hunk 1]
```diff
@@ -59,6 +61,7 @@
 	'phone'     => true,
 	'organizer' => true,
 ];
+$view_more_link = '#';
```
Comment (tr1b0t) [inline src/views/v2/widgets/events-list.php:64]: :warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 73e200639cd2e31614ea3c32858022d1</sub>

File: src/views/v2/widgets/events-list.php [hunk 2]
```diff
@@ -92,10 +95,20 @@
 
 				<div class="tribe-events-widget-events-list__events">
 					<?php foreach ( $events as $event ) : ?>
-						<?php $this->template( 'widgets/events-list/event', [ 'event' => $event, 'display' => $display ] ); ?>
+						<?php
+						$this->template(
+							'widgets/events-list/event',
+							[
+								'event' => $event,
```
Comment (tr1b0t) [inline src/views/v2/widgets/events-list.php:102]: :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'event'" and double arrow, but found 1.

<sub>hash: 023a842d578e93e9bd33a3d8a0d393a7</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

**Review by bordoni — APPROVED** [review]

File: src/views/v2/widgets/events-list/view-more.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post $view_more_link The URL to view all events.
```
Comment (lucatume) [inline src/views/v2/widgets/events-list/view-more.php:14]: This seems like a copy-and-pasta issue:

```suggestion
 * @var string $view_more_link The URL to view all events.
```

**Review by lucatume — APPROVED** [review]
A smaller change in the doc blocks, then good to go.

File: src/views/v2/widgets/events-list/view-more.php (hunk 3 above)
Comment (paulmskim) [reply src/views/v2/widgets/events-list/view-more.php:14]: you're right, thanks.

**Pull request merged**